asciinema2gif

```diff
@@ -5,18 +5,25 @@
 # Halt on error.
 set -e
 
-rundir=$PWD
+readonly program="$(basename "${0}")"
+readonly rundir="$(cd "$(dirname "${0}")" && pwd -P)"
+
+# Show instructions when using wrong arguments
+if [[ -z "${1}" ]] || ! [[ "${1}" = 'https://asciinema.org/a/'* ]]; then
```

Comment:
Will fire when given with no arguments, or when the argument isn’t an asciinema url.